.pre-commit-config.yaml

```diff
@@ -0,0 +1,21 @@
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v2.3.0
```

Comment:
v4.3.0 can be used now.

If you are using pre-commit consider adding flake8:

```
  - repo: https://github.com/PyCQA/flake8
    rev: 4.0.1
    hooks:
      - id: flake8
```

Reply:
Version updated, and flake8 added.